mudirac/mudirac.xml

```diff
@@ -0,0 +1,74 @@
+<tool id="mudirac" name="MuDirac" version="@TOOL_VERSION@+galaxy@WRAPPER_VERSION@" python_template_version="3.5" profile="23.0" license="MIT">
+    <description> Muonic atom Dirac equation solver </description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.0.1</token>
+        <token name="@WRAPPER_VERSION@">0</token>
+    </macros>
+    <creator>
+        <person givenName="Subindev" familyName="Devadasan" url="https://github.com/subindev-d" identifier="https://orcid.org/0009-0001-1723-4394"/>
+        <organization url="https://muon-spectroscopy-computational-project.github.io/index.html" name="The Muon Spectroscopy Computational Project"/>
+    </creator>
+    <requirements>
+        <requirement type="package" version="1.0.1">mudirac</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    cat $input_yaml > input.in &&
+    mudirac input.in
+    ]]></command>
+    <configfiles>
+        <configfile name="input_yaml">element: $element
+isotope: $isotope
+xr_lines: $xr_lines
+uehling_correction: $uehling_correction
+nuclear_model: $nuclear_model
+#if $electronic_config:
+electronic_config: $electronic_config
+#end if</configfile>
+    </configfiles>
+    <inputs>
+        <param name="element" type="text" value="H" label="Element" help="Determines the nuclear charge. Can be any symbol in the periodic table up to Z=111, Roentgenium (Rg)" optional="false"/>
```

Comment:
Would be a select with all symbols better and safer for the user?

Reply:
Did this for another (currently unreleased) prototype tool:
https://github.com/MaterialsGalaxy/soprano-tools/blob/c06d72485cc03e2866b7f061766e0d28d4444e1c/soprano_plotnmr/soprano_plotnmr.xml#L7-L133